python/tvm/relay/op/contrib/arm_compute_lib.py

```diff
@@ -201,38 +201,32 @@ def check_avg_pool2d(extract):
         pool = extract.args[0]
         if pool.args[0].attrs.dtype != "int32":
             return False
-        return avg_pool2d(pool.attrs, pool.args, from_quantized_composite=True)
+        return avg_pool2d(pool, from_quantized_composite=True)
 
     def check_l2_pool2d(extract):
         """Check l2 pool2d pattern is supported by ACL."""
         pool = extract.args[0]
-        return avg_pool2d(pool.attrs, pool.args)
+        return avg_pool2d(pool)
 
     return [
         ("arm_compute_lib.conv2d", conv_pattern(), check_conv),
         ("arm_compute_lib.qnn_conv2d", qnn_conv_pattern(), check_qnn_conv),
         ("arm_compute_lib.dense", dense_pattern(), check_dense),
         ("arm_compute_lib.qnn_dense", qnn_dense_pattern(), check_qnn_dense),
-        ("arm_compute_lib.qnn_conv2d", qnn_conv_pattern(), check_qnn_conv),
```

Comment:
Unrelated?

Reply:
There was a duplication here (L213)

Reply:
Again, doesn't relate to this change so should be in a separate fix-up patch.

Reply:
updated

Reply:
I can still see the change here

Reply:
Done. Duplication reverted.